themes/custom/ts_wrin/sass/global/_wysiwyg.scss

```diff
@@ -398,3 +398,55 @@ table {
     height: 100%;
   }
 }
+
+.report-teaser {
+  @extend .secondary;
+  border-left: none;
+  padding: 20px 0 12px;
+  margin-top: $space-xs;
+
+  & p:first-child {
+    margin-top: 0;
+  }
+
+  &.alignleft p.secondary.body-link,
+  &.alignright p.secondary.body-link {
+    @include mq($md) {
+      &:first-of-type {
+        margin-top: 0;
+      }
+    }
+  }
+  img {
+    max-height: 240px;
+    padding-bottom: $space-xs;
+    width: 180px;
+  }
+
+  p {
+    @include font($acumin-semi-cond, "semi-bold");
+    @include font-line-height(18, 26);
+  }
+  figure {
+    margin: 0 0 20px 0;
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    align-items: center;
+    display: flex;
+
+    .media--type-image {
+      padding-right: $space-xs;
+    }
+  }
+  &.alignright {
+    @include mq($lg) {
+      display: inline;
+      float: right;
+      margin: 0 -25.7% 0 $space-sm;
```

Comment:
I'm going to change this slightly and make it a fixed-width of 180px then float it to the right by its width+the grid gutter+the margin: `230px`. that sticks to the grid gutter nicely (you can test it by showing the grid with a `\?show-grid=1` in the url if you need to.

Otherwise this looks great and I'm merging!